integrations-and-supported-tools/plotly/notebooks/Neptune_Plotly_Support.ipynb

```diff
@@ -1,266 +1,266 @@
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "XSHLD6AkI7LW"
-      },
-      "source": [
-        "# Step 4: Stop logging\n",
-        "\n",
-        "<font color=red>**Warning:**</font>  \n",
-        "Once you are done logging, you can stop tracking the run using the `stop()` method.\n",
-        "This is needed only while logging from a notebook environment. While logging through a script, Neptune automatically stops tracking once the script has completed execution."
-      ]
-    },
-    {
-      "cell_type": "code",
-      "metadata": {
-        "id": "Zgh3NwDoJAuG"
-      },
-      "source": [
-        "run.stop()"
-      ],
-      "execution_count": null,
-      "outputs": []
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "had9MNRtQ7Mr",
-        "tags": [
-          "comment"
-        ]
-      },
-      "source": [
-        "# Conclusion\n",
-        "\n",
-        "You’ve learned how to:\n",
-        "\n",
-        "* Initialize Neptune and create a run,\n",
-        "* Log `plotly` charts to Neptune."
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "_3ibcYhPQ7Mr",
-        "tags": [
-          "comment"
-        ]
-      },
-      "source": [
-        "# What's next\n",
-        "\n",
-        "Now that you know how to log charts to Neptune, you can learn:\n",
-        "\n",
-        "* [How to log other types of metadata to Neptune](https://docs.neptune.ai/you-should-know/logging-and-managing-runs-results/logging-runs-data#what-objects-can-you-log-to-neptune)\n",
-        "* [How to download runs data from Neptune](https://docs.neptune.ai/user-guides/logging-and-managing-runs-results/downloading-runs-data)\n",
-        "* [How to connect Neptune to the ML framework you are using](https://docs.neptune.ai/essentials/integrations)"
-      ]
-    }
-  ]
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.7"
+  },
+  "toc-autonumbering": true,
+  "toc-showcode": false,
+  "toc-showmarkdowntxt": false,
+  "toc-showtags": true
+ },
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "GG3_2HyRVY4Y",
+    "tags": [
+     "header",
+     "comment"
+    ]
+   },
+   "source": [
+    "# Introduction\n",
+    "\n",
+    "This guide will show you how to:\n",
+    "\n",
+    "* Initialize Neptune and create a `run`,\n",
+    "* Log `plotly` charts to Neptune."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "1tBfsYCm05rl"
+   },
+   "source": [
+    "# Step 0: Before you start\n",
+    "\n",
+    "Install `neptune-client` and `plotly`\n",
```

Comment:
@SiddhantSadangi the requirements here should have explicit versions so that people know what works.
We'll be updating versions periodically.